internal/task/executor.go

```diff
@@ -68,5 +70,8 @@ func runTaskSafely(ctx context.Context, t Task, resolver file.Resolver, s sbomsy
 		}
 	}()
 
-	return t.Execute(ctx, resolver, s)
+	start := time.Now()
+	res := t.Execute(ctx, resolver, s)
+	log.WithFields("task", t.Name(), "elapsed", time.Since(start))
```

Comment:
This needs to have a log directive after the `.WithFields()` call, e.g.:

```
log.WithFields("task", t.Name(), "elapsed", time.Since(start)).Info("task completed")
```

Reply:
fixed